src/classes/Auth/Auth.ts

```diff
@@ -96,11 +98,13 @@ export class Auth extends PassageBase {
                 appId: this.config.appId,
                 createMagicLinkRequest: {
                     ...args,
+                    // eslint-disable-next-line camelcase
+                    user_id: userId,
```

Comment:
it was being passed as part of the `...args` input, but the codegen wants [this instead](https://github.com/passageidentity/passage-node/blob/742dabf692e50c020d45f57df8747d84738fd168/src/generated/models/CreateMagicLinkRequest.ts#L96)